types/gitconfiglocal/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for gitconfiglocal 2.0
+// Project: https://github.com/soldair/node-gitconfiglocal#readme
+// Definitions by: Florian Keller <https://github.com/ffflorian>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare namespace gitconfig {
+    interface GitConfig {
+        [key: string]: GitConfig;
```

Comment:
```suggestion
        [key: string]: any;
```
The current interface doesn't allow properties to be primitives (numbers, strings, etc.). Is this intended?

Reply:
Ah, you are right. Will fix it :)